src/Common/test-runtime/project.json

```diff
@@ -18,6 +18,7 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00520-02",
     "System.Collections.NonGeneric": "4.0.1",
     "System.Collections.Specialized": "4.0.1",
+    "System.Net.WebSockets.Client": "4.0.1-beta-24322-03",
```

Comment:
Can we try to not add `"System.Net.WebSockets.Client": "4.0.1-beta-24322-03"` in test-runtime as this will add this package dependency to ALL test project while we likely only need this for websocket tests?